doc/classes/PopupMenu.xml

```diff
@@ -409,6 +436,7 @@
 			<argument index="1" name="icon" type="Texture">
 			</argument>
 			<description>
+				Sets/replaces the icon [Texture] of the specified [code]idx[/code]
```

Comment:
Don't use two verbs with a slash in between.
Suggestion: `Replaces the [code]icon[/code] of the item at index [code]idx[/code]`.